### DISCUSSION TITLE
jdk11 or jdk17 support?

### DISCUSSION THREAD

**Discussion by kterusaki** [stanfordnlp/CoreNLP#1432] [General]
Our company is shifting to move towards jdk11 / jdk17. Unfortunately, there's a known issue with using jdk11 and jdk17 that's documented [here](https://github.com/stanfordnlp/CoreNLP/discussions/1335). Are there current discussions or a timeline for supporting newer jdk versions?

**Comment by AngledLuffa** [comment]
It's running fine for me on Java 11.  There's a plan to move to Java 11 as the default by the end of the summer

https://github.com/stanfordnlp/CoreNLP/discussions/1429

**Reply by kterusaki** [reply]
thanks for clarifying! I was on an older version `3.8.0` which may have been incompatible with jdk11. Once I upgrade to `3.9.2` I was able to get things to work.

Also, good to know about the move to java 11!

**Comment by manning** [comment]
Updated timeline: 2025 is the year we will move to Java 11.
